tests/gen_cert_test.py

```diff
@@ -181,3 +216,70 @@ def test_cert_date(self, force_date, locale, timezone, expected):
         """
         cert_date = get_cert_date(force_date, locale, timezone)
         assert_equal(expected, cert_date)
+
+
+class CertificateSubtemplateTests(unittest.TestCase):
+    """
+    Tests the subtemplates feature for certs
+
+    Subtemplates are currently written to key on designations only
+
+    These tests have a specific order but do not need to be run in order.
+    They were split and ordered to more easily identify errors while testing.
```

Comment:
Holy moly man, I did not read this, not even the TL;DR.  

How many comments do we need for this comment?  I need to take my shoes off to count.  I think there are more lines talking about this comment then lines in the test file.  Quite possibly the platform.